modules/openid_auth/app/controllers/openid_auth/v0/validation_controller.rb

```diff
@@ -1,18 +1,27 @@
 # frozen_string_literal: true
 
 require_dependency 'openid_auth/application_controller'
+require 'mvi/errors/errors'
 
 module OpenidAuth
   module V0
     class ValidationController < ApplicationController
       def index
-        render json: validated_payload, serializer: OpenidAuth::ValidationSerializer
-      rescue StandardError => e
-        raise Common::Exceptions::InternalServerError, e
+        raise StandardError unless @current_user.icn
+        render json: validated_payloa, serializer: OpenidAuth::ValidationSerializer
```

Comment:
`validated_payloa` -> `validated_payload`

Reply:
sorry last minute rubocop fix that messed it up, I'll fix it now lol